store/sqlstore/upgrade.go

```diff
@@ -936,18 +937,61 @@ func upgradeDatabaseToVersion529(sqlStore SqlStore) {
 		if _, err := sqlStore.GetMaster().Exec(updateThreadChannelsQuery); err != nil {
 			mlog.Error("Error updating ChannelId in Threads table", mlog.Err(err))
 		}
+	}
 
+	if shouldPerformUpgrade(sqlStore, VERSION_5_28_1, VERSION_5_29_0) {
 		saveSchemaVersion(sqlStore, VERSION_5_29_0)
 	}
 }
 
-func upgradeDatabaseToVersion530(sqlStore SqlStore) {
-	// if shouldPerformUpgrade(sqlStore, VERSION_5_29_0, VERSION_5_30_0) {
-
-	sqlStore.CreateColumnIfNotExistsNoDefault("FileInfo", "Content", "longtext", "text")
+func hasMissingMigrationsVersion529(sqlStore SqlStore) bool {
+	scIdInfo, err := sqlStore.GetColumnInfo("SidebarCategories", "Id")
+	if err != nil {
+		mlog.Error("Error getting column info for migration check",
```

Comment:
Gentle reminder to check our logging [principles](https://developers.mattermost.com/contribute/server/style-guide/#log-levels) before merging this change.